### PR TITLE
avr-rss2 platform fixes

### PR DESCRIPTION
**Pull request by herjulf**
avr-rss2 platform. Serial line input bugfix. Misc. clean-up. Private sniffer app removed.

### CODE REVIEW

File: platform/avr-rss2/contiki-main.c [hunk 1]
```diff
@@ -339,8 +350,8 @@ initialize(void)
   NETSTACK_NETWORK.init();
 
 #if ANNOUNCE_BOOT
-  PRINTA("MAC=%s, RDC=%s, NETWORK=%s, channel=%-u, check-rate-Hz=%-u, tx-power=%-u\n", NETSTACK_MAC.name,
-         NETSTACK_RDC.name, NETSTACK_NETWORK.name, rf230_get_channel(),
+  PRINTA("PAN=0x%X, XMAC=%s, RDC=%s, NETWORK=%s, channel=%-u, check-rate-Hz=%-u, tx-power=%-u\n", rf230_get_panid(),
```
Comment (nvt) [inline platform/avr-rss2/contiki-main.c]: There appears to be a typo here with "XMAC=%s".

Comment (herjulf) [inline platform/avr-rss2/contiki-main.c]: Hi,
Good spotting. Thanks.

**Comment by nvt** [comment]
:+1:

**Comment by herjulf** [comment]
Gott!

**Pull request merged**